tools/hw_decoder_bench.py

```diff
@@ -21,6 +21,8 @@
 parser = argparse.ArgumentParser(description='DALI HW decoder benchmark')
 parser.add_argument('-b', dest='batch_size', help='batch size', default=1, type=int)
 parser.add_argument('-d', dest='device_id', help='device id', default=0, type=int)
+parser.add_argument('-n', dest='gpu_num',
+                    help='Number of GPUs used starting from device_id', default=1, type=int)
```

Comment:
I recall some issue that `device_ids` do not have to be consecutive (I might be wrong though). How about providing a list of devices instead? Something along the lines:
```suggestion
parser.add_argument('-d', dest='device_id', help='device_id', default=0, nargs='*')
```
And the usage would be:
```bash
python hw_decoder_bench.py -d 1 3 -b ...
python hw_decoder_bench.py -d 0 -b ...
```
```python
    for di in range(args.device_id):
        pipes.append(DecoderPipeline(device_id=di))
```

Reply:
The numbers are consecutive, but the order may differ from the PCI order.
I rather imagine this script to be used to test on heterogenous systems and the user will just decide how many GPUs is willing to use.